locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
Good to know; but I wonder if that's always the intent of the localizers. (Sometimes the terms are actually removed, e.g. `zn-CN` and `zh-TW` with most of the ordinals.)